handlers/handlers.go

```diff
@@ -4,46 +4,57 @@ import (
     "net/http"
     "encoding/json"
     "fmt"
-    "log"
+    "sync"
+    "github.com/julienschmidt/httprouter"
     "github.com/skrzepto/IPRO-VAULT-SERVER/models"
-    _ "github.com/jinzhu/gorm/dialects/sqlite"
-    "github.com/jinzhu/gorm"
 )
 
 type Impl struct {
-    DB *gorm.DB
+  sd_map map[string]models.SensorData
```

Comment:
Maybe rename it to sd_table?

I believe it is more generic and the name could stay the same if we changed the implementation to an array or a database table it would stay the same. I understand the benefits of the Hungarian notation but Go has types. If you decide to leave it like this though, I'm fine either way.